salt/modules/virt.py

```diff
@@ -5394,7 +5394,8 @@ def _get_all_volumes_paths(conn):
 
     :param conn: libvirt connection to use
     '''
-    volumes = [vol for l in [obj.listAllVolumes() for obj in conn.listAllStoragePools()] for vol in l]
+    volumes = [vol for l in
+                [obj.listAllVolumes() for obj in conn.listAllStoragePools() if obj.info()[0] == 2] for vol in l]
```

Comment:
Could you please consider to use the libvirt constant instead of hardcoding the value here?
```
obj.info()[0] == libvirt.VIR_STORAGE_POOL_RUNNING
```

Reply:
done